src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts

```diff
@@ -28,7 +28,7 @@ export default () => `
 				<div class="section deploy">
 					<h2 class="caption">${escape(localize('welcomePage.deploy', "Deploy"))}</h2>
 					<ul>
-						<li><a href="command:azdata.resource.deploy">${escape(localize('welcomePage.newDeployment', "New Deployment…"))}</a></li>
+						<li><a href="command:resourceDeployment.new">${escape(localize('welcomePage.newDeployment', "New Deployment…"))}</a></li>
```

Comment:
Instead of having these hardcoded here like this what if we had these be contributable as well by extensions and then do what Anthony suggested and have the extension activate on dashboard open?

Reply:
I've gotten a number of different options here that nobody seems to be super excited about 😄.

If I just change the extension to load when the welcome page is opened then, any objections, @Charles-Gagnon, @anthonydresser ?

Reply:
Yeah - if that works I say we go for that. Seems the simplest way to handle the situation. 

(my suggestion is definitely not something I think we should necessarily do, was just thinking aloud)

Reply:
activate the extensions on welcome page open? I don't see an existing option for activationEvent.